### PR TITLE
Resolve @typescript-eslint/no-unnecessary-condition warnings for static/panes/

### PR DESCRIPTION
**Pull request by jeremy-rifkin**

### CODE REVIEW

**Review by RubenRBS — APPROVED** [review]
All seems good!. Need to test it locally later today before mering (Not that I don't trust you though!)

**Comment by jeremy-rifkin** [comment]
Thanks! An extra set of eyes on testing is very much appreciated :)

**Comment by mattgodbolt** [comment]
How's this looking, @RubenRBS what do you think?

File: static/panes/ast-view.ts [hunk 1]
```diff
@@ -120,18 +120,20 @@ export class Ast extends MonacoPane<monaco.editor.IStandaloneCodeEditor, AstStat
...
+                const hoverCode = this.astCode[e.target.position.lineNumber - 1];
                 let sourceLine = -1;
                 let colBegin = -1;
                 let colEnd = -1;
                 // We check that we actually have something to show at this point!
+                /* eslint-disable-next-line @typescript-eslint/no-unnecessary-condition */ // TODO
                 if (hoverCode.source && hoverCode.source.from) {
                     sourceLine = hoverCode.source.from.line;
                     // Highlight part of a line corresponding to the node if it fits on one line
+                    /* eslint-disable-next-line @typescript-eslint/no-unnecessary-condition */ // TODO
```
Comment (RubenRBS) [inline static/panes/ast-view.ts:136]: Here, the types are definetely wrong as both from and to can be null

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**